app/edge/components/groups-datatable/groupsDatatable.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+
+angular.module('portainer.edge').component('edgeGroupsDatatable', {
+  templateUrl: './groupsDatatable.html',
+  controller: 'EdgeGroupsDatatableController',
```

Comment:
Why do we need a specific controller for this datatable instead of using the Generic controller?

Reply:
no need